src/mpriscontroller.cpp

```diff
@@ -152,10 +212,10 @@ bool MprisController::next()
         return false;
     }
 
-    QDBusPendingReply<> reply = m_mprisPlayerInterface->Next();
+    QDBusPendingReply<> reply = priv->m_mprisPlayerInterface.Next();
     QDBusPendingCallWatcher *watcher = new QDBusPendingCallWatcher(reply, this);
-    QObject::connect(watcher, SIGNAL(finished(QDBusPendingCallWatcher*)),
-                     this, SLOT(onFinishedPendingCall(QDBusPendingCallWatcher*)));
+    QObject::connect(watcher, &QDBusPendingCallWatcher::finished,
+                     priv, &MprisControllerPrivate::onFinishedPendingCall);
```

Comment:
Old thing, but these three lines get repeated a bunch of times. Could combine to some helper method.